### PR TITLE
Adr2/update report html UI

### PR DESCRIPTION
**Pull request by GitVitor**
## Description
UI change:
    1. add Gotham Font Family;
    2. Variables CSS with colors;
            2.1. Update red color to use: #f14668;
            2.2. Update green color to use: #48c78e;
    3. Add visual wrapper to following areas: request content, request response and request test;
    4. New endpoint status UI;
 
## Motivation behind this PR?
Following recent discussions on #161 we decide to implement new details to help users on visualize related request data.

## What type of change is this?
Feature

## Checklist

- [x]  I have added a changelog entry / my PR does not need a new changelog entry. [Instructions](https://github.com/scanapi/scanapi/wiki/Changelog).
- [ ] I have added/updated unit tests. [Instructions](https://github.com/scanapi/scanapi/wiki/Writing-Tests).
- [x] New and existing unit tests pass locally with my changes. [Instructions](https://github.com/scanapi/scanapi/wiki/Run-ScanAPI-Locally#tests)
- [ ] I have self-documented code my changes by adding docstring(s) and comment(s). [Instructions](https://github.com/scanapi/scanapi/wiki/First-Pull-Request#7-make-your-changes)
- [x] Current PR does not significantly decrease the code coverage and docstring coverage.
- [x] My code follows the style guidelines of this project.
- [x] I have run ScanAPI locally and manually tested my changes. [Instructions](https://github.com/scanapi/scanapi/wiki/Run-ScanAPI-Locally).
- [x] I have squashed my commits. [Instructions](https://github.com/scanapi/scanapi/wiki/Squashing-Commits).

## Issue

 #161

### CODE REVIEW

**Comment by camilamaia** [comment]
Hey, @GitVitor, thank you very much for the PR 🙌

I run the branch on my local machine and tested it using Chrome, Firefox, and Safari.

I made a circle in the improvements points I found. In Safari, there is some information missing, for example. There are other small issues related to alignment, space...

#### Chrome
Version 93.0.4577.63 (Official Build) (x86_64)

Chrome
chrome-extended

#### Firefox
91.0 (64-bit)

firefox
firefox-expanded

#### Safari
Version 14.1.2 (16611.3.10.1.6)

safari
safari-expanded

**Comment by Pradhvan** [comment]
@GitVitor hey 👋🏾  are you still working on this ?

**Comment by GitVitor** [comment]
Hello folks, sorry i was busy in last month. I will work on this in next days.

**Review by astenstrasser — CHANGES_REQUESTED** [review]
Hi @GitVitor, thanks for the PR!
I run it locally and in chrome looks like somethings are not aligned, like status and tables.
I tried to check what was the problem but could not fix it. Can you take a look?
Thanks!!

image

File: scanapi/templates/report.html [hunk 1]
```diff
@@ -9,18 +9,36 @@
...
+    <link href="http://fonts.cdnfonts.com/css/gotham" rel="stylesheet">
```
Comment (astenstrasser) [inline scanapi/templates/report.html:12]:
```suggestion
```

File: scanapi/templates/report.html [hunk 2]
```diff
@@ -9,18 +9,36 @@
...
+        
         html,
         body {
             margin: 0;
             padding: 0;
             background-color: #eeeeee;
             font-size: 18px;
-            font-family: -apple-system, system-ui, BlinkMacSystemFont, "Segoe UI", Roboto,
-                "Helvetica Neue", Arial, sans-serif;
+            font-family: 'Gotham', sans-serif;
```
Comment (astenstrasser) [inline scanapi/templates/report.html:41]:
```suggestion
            font-family: -apple-system, system-ui, BlinkMacSystemFont, "Segoe UI", Roboto,
                 "Helvetica Neue", Arial, sans-serif;
```

**Review by astenstrasser — APPROVED** [review]

File: CHANGELOG.md [hunk 3]
```diff
@@ -6,6 +6,9 @@ The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
 and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
 ## [Unreleased]
+### Changed
+- Implement new details to help users on visualize related request data.
+-
```
Comment (astenstrasser) [inline CHANGELOG.md:11]:
```suggestion

```

File: CHANGELOG.md [hunk 4]
```diff
@@ -6,6 +6,8 @@ The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
 and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
 ## [Unreleased]
+### Changed
+- Implement new details to help users on visualize related request data.
 ### Fixed
```
Comment (astenstrasser) [inline CHANGELOG.md:11]:
```suggestion

### Fixed
```

**Pull request merged**